book/02-git-basics/sections/recording-changes.asc

```diff
@@ -334,10 +334,10 @@ index 0000000..03902a1
 +My Project
 ----
 
-It's important to note that `git diff` by itself doesn't show all changes made since your last commit -- only changes that are still unstaged.
-If you've staged all of your changes, `git diff` will give you no output.
+Ito ay importanteng i-note na ang `git diff` na sa sarili niya ay hindi nagpapakita ng lahat ng mga pagbabago na iyong nagawa sa huli mong commit -- mga pagbabago lamang na hindi pa rin naitanghal.
```

Comment:
you could change `Ito ay importanteng i-note` to `Importanteng tandaan`

Reply:
done change